internal/handlers/pg/msg_createindexes.go

```diff
@@ -166,6 +166,16 @@ func (h *Handler) MsgCreateIndexes(ctx context.Context, msg *wire.OpMsg) (*wire.
 				return err
 			}
 
+			if index.Unique != nil {
+				// when non-nil value is specified for unique, command response contains index count
```

Comment:
Not only, see my PR https://github.com/FerretDB/FerretDB/pull/2936/files#diff-57e5c4a5a672cdfe4ab09fc83c250dea67bfeae45f4618963fbc13a2a5c5761b (I added some more tests) :)

Reply:
:eyes: yes indeed.